west.yml

```diff
@@ -139,7 +139,11 @@ manifest:
       revision: c39888ff74acf421eeff9a7514fa9b172c3373f7
     - name: trusted-firmware-m
       path: modules/tee/tfm
-      revision: 4544ab9fc4d5305ae5de97c73a9356ef0f342f7a
+      revision: pull/37/head
+    - name: tfm-mcuboot
```

Comment:
We should add a comment here about what is going in with `tfm-mcuboot` and `trusted-firmware-m` since this module isn't used by zephyr directly.

Reply:
Added